GAMMA: 2.2
LOW_BOUND: 500
UP_BOUND: 1500
